lib/go-tc/topologies.go

```diff
@@ -47,3 +47,25 @@ type TopologiesResponse struct {
 	Response []Topology `json:"response"`
 	Alerts
 }
+
+// TopologiesQueueUpdateRequest encodes the request data for the POST
+// topologies/{{name}}/queue_update endpoint.
+type TopologiesQueueUpdateRequest struct {
+	Action string `json:"action"`
+	CDNID  int64  `json:"cdnId"`
+}
+
+// TopologiesQueueUpdate encodes the response data for the POST
```

Comment:
Should be `TopologiesQueueUpdateResponse`

Reply:
Changed to `TopologiesQueueUpdateResponse` in a3aba0f95c